### ISSUE TITLE
error: bundling failed: Error: Unable to resolve module `./src/Reels/components/Reels` from `node_modules/react-native-instagram-reels/index.js`: 

### ISSUE THREAD

**Issue by hgvansh-him** [kartikeyvaish/react-native-instagram-reels#2]
error: bundling failed: Error: Unable to resolve module `./src/Reels/components/Reels` from `node_modules/react-native-instagram-reels/index.js`:  ios issue

**Comment by KHALIL965** [comment]
Yeah I am also facing this issue any solutions I have installed the  library but its throwing the same error.

**Comment by KrushnaNaghate** [comment]
ok, I sorted it out I just got the same error, go to directory src>reels>component> and rename all files from .jsx to .js, it will sort out the issue

**Comment by bakoserge** [comment]
Any solution for this? or is there an alternative to this package

**Comment by reachmesagar** [comment]
> Any solution for this? or is there an alternative to this package

go to the react-native-instagram-reels package , and change all the jsx packages to js extension  . and it will works .
